### DISCUSSION TITLE
Option to exclude some books from average reading time statistic

### DISCUSSION THREAD

**Discussion by Sera-Moon** [mateusz-bak/openreads#716] [Ideas]
**Is your feature request related to a problem? Please describe.**
I not only use the App to track my fictional books but also non-fiction books like Cookbooks or Textbooks/Studybooks. These books take some time to finish (for example, the cookbook 50 Potato Recipes will take several weeks to finish; most of my textbooks for school even take nearly a year to finish) and increases the average reading time in the statistics to several weeks/months, although my average reading time for my fictional books would only be a few days. 

**Describe the solution you'd like**
I'd like an option to exclude these books from my reading time statistics. For example, a drop-down menu like this

*Include into reading time statistics?*
Yes (default)
No
(You have to actively open the drop-down to exclude the book)

An option is to add a second average reading time statistic: 
Average reading time (all books)  --> included + excluded
Average reading time (without excluded books) 

**Describe alternatives you've considered**
A toggle to switch it on/off in the edit page of the book. But there may be the risk to accidentally exclude from/include to the average time statistics

**Comment by joers2** [comment]
Hi there! My name is Jinte and i am a coding student from the Netherlands. Currently i have an assignment where i am supposed to help with an open source project< and i would love to work on this feature could i be assigned to it?
